3p/integration.js

```diff
@@ -56,6 +56,7 @@ import {criteo} from '../ads/criteo';
 import {dotandads} from '../ads/dotandads';
 import {doubleclick} from '../ads/google/doubleclick';
 import {eplanning} from '../ads/eplanning';
+import {fakead3p} from '../ads/fakead3p';
```

Comment:
This gets delivered to production! We need to only register this in the testing environments.